mscore/musescore.cpp

```diff
@@ -1,4 +1,4 @@
-//=============================================================================
+﻿//=============================================================================
```

Comment:
Just don't make that chunk part of the commit? Pretty easy in GitHub for Windows, just deselect that part of the changes, then commit, then ignore that remainder. Not sure how this works with git commandline though.